spec/unit/blockscore/collection/member_spec.rb

```diff
@@ -1,7 +1,32 @@
 module BlockScore
   class Collection
     RSpec.describe Member do
-      describe '.save' do
+      describe '#save' do
+        let(:question_set) { QuestionSet.new }
+        subject(:member) { described_class.new(person, question_set) }
+
+        context 'unsaved person' do
+          let(:person) { BlockScore::Person.new(attributes_for(:person)) }
+
+          it 'saves the person as part of member save' do
+            expect(person).to_not be_persisted
+            expect(member.save).to be true
+            expect(person).to be_persisted
+          end
+        end
+
+        context 'previously saved person' do
+          let(:person) { BlockScore::Person.create(attributes_for(:person)) }
+
+          it 'contains the proper fields' do
+            expect(member.save).to be true
+            expect(person).to be_persisted
+            expect(person.attributes.fetch(:question_sets)).to include question_set.id
+            expect(person.attributes.fetch(:question_sets).size).to be 1
+            expect(member).to be_an_instance_of described_class
+            expect(member.person_id).to eql person.id
```

Comment:
@dkubb I thought `be` was equivalent to `equal`. That won't work with a string correct?